internal/cli/command_factory.go

```diff
@@ -76,11 +76,15 @@ func (factory *CommandFactory) Build(command CommandDefinition) *cobra.Command {
 				os.Exit(1)
 			}
 
-			factory.telemetryService = telemetry.NewService(
+			telemetryService := telemetry.NewService(
 				factory.profile.telemetryMode,
 				factory.profile.User().PublicAPIKey,
+				factory.profile.telemetryWriteKey,
```

Comment:
can remove this, we are going to store the write key as a `var` in the `telemetry` package